### ISSUE TITLE
Which license

### ISSUE THREAD

**Issue by NisJorgensen** [DemocracyEarth/paper#171]
The LICENSE file contains the MIT license (which talks about "software"). The paper itself states that it is released under CC-BY-NC-SA. So which one is it?

**Comment by santisiri** [comment]
True. I checked to include CC license on Github but as far as I could see it's not an available option, only offers software related licenses. So in the sense of the code of the document (mediawiki) -which after all is a piece of software- i think that MIT covers it well. On the PDF since its printed and not open to instant modifications, a CC license seemed more suitable.

**Comment by bhack** [comment]
Can [copyright tags](https://m.mediawiki.org/wiki/Special:MyLanguage/Manual:Configuration_settings#Copyright) be used?

**Comment by santisiri** [comment]
@bhack I didn't know those existed. Yes, we should definitely include the Creative Commons license we used on the PDF version then.
